spec/jobs/clean_up_uploads_spec.rb

```diff
@@ -305,4 +305,14 @@ def fabricate_upload(attributes = {})
     expect(Upload.exists?(id: expired_upload.id)).to eq(false)
     expect(Upload.exists?(id: badge_image.id)).to eq(true)
   end
+
+  it "deletes external upload stubs that have expired" do
+    Fabricate(:external_upload_stub, status: ExternalUploadStub.statuses[:created], created_at: 10.minutes.ago)
+    Fabricate(:external_upload_stub, status: ExternalUploadStub.statuses[:created], created_at: (ExternalUploadStub::CREATED_EXPIRY_HOURS.hours + 10.minutes).ago)
+    Fabricate(:external_upload_stub, status: ExternalUploadStub.statuses[:uploaded], created_at: 10.minutes.ago)
+    Fabricate(:external_upload_stub, status: ExternalUploadStub.statuses[:uploaded], created_at: (ExternalUploadStub::UPLOADED_EXPIRY_HOURS.hours + 10.minutes).ago)
+    expect(ExternalUploadStub.count).to eq(4)
+    Jobs::CleanUpUploads.new.execute(nil)
+    expect(ExternalUploadStub.count).to eq(2)
```

Comment:
We need to be careful here. Asserting for the count is usually not enough. We could be deleting the wrong records and can still end up with a count of `2`.

Reply:
Yes good point, will fix